CHANGELOG.md

```diff
@@ -1,19 +1,28 @@
 # Changelog
 
 ### Unreleased
+
+
+# v12.0
+
+Breaking note: this change passes all non Kubetools specific container config through to the generated K8s container spec. Any invalid/unused data would have previously been ignored will now be passed to K8s and throw an error.
+
+- Pass any non-Kubetools specific container config through to K8s container spec
 - Implicitly create target namespace if it does not exist
 - The `--cleanup` flag will now remove remaining empty namespaces
```

Comment:
Mind updating this for clarity to be more specifically _the target namespace_, not any empty namespaces please?

Reply:
ty @Fizzadar!